secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -144,6 +151,7 @@ private static Secret createSecret() throws IOException {
                         Replication.newBuilder()
                             .setAutomatic(Replication.Automatic.newBuilder().build())
                             .build())
+                    .putLabels(LABEL_KEY, LABEL_VALUE)
```

Comment:
Why do we need to the add the labels here?

Reply:
So that in ViewSecretLabels, there are labels in the secret.